src/components/Calendar/src/Calendar.vue

```diff
@@ -81,30 +79,20 @@
 import {
 	startOfWeek,
 	startOfMonth,
-	eachDayOfInterval,
-	getDay,
-	lastDayOfMonth,
+	endOfMonth,
 	addDays,
-	format,
 	isToday,
-	isAfter,
```

Comment:
Removed in favor of using larger/smaller than operators for date comparison.

Reply:
Is removing a lot of these methods where the majority of the file savings came from?

Reply:
Yep!